### PR TITLE
Bottom oxygen maps.

### PR DESCRIPTION
**Pull request by m-n-o**
Add functions `plot_bottom_oxygen()` and `record_bottom_oxygen()` in output.jl.
Add bottom mask evaluation in plot.jl.

### CODE REVIEW

File: app/varna/plot.jl [hunk 1]
```diff
@@ -26,6 +26,24 @@ times = T.times
...
 
+
+# bottom_z evaluation
+bottom_z = ones(Int, size(O₂, 1), size(O₂, 2))
+for i in 1:size(O₂, 1)
+    for j in 1:size(O₂, 2)
+        for k in size(O₂, 3):-1:1  # Loop backwards to find the latest non-zero
+            if O₂[i, j, k, 1] == 0
+                bottom_z[i, j] = k
+                if k != 12
```
Comment (fisa47) [inline app/varna/plot.jl]: Why k != 12? Should be Nz?

File: src/output.jl [hunk 2]
```diff
@@ -56,6 +56,29 @@ function record_surface_speed(u, v, Nz, times, folder; colorrange = (0, 0.5), co
     end
 end
 
+function record_bottom_oxygen(O₂, times, bottom_z, folder; colorrange = (-1, 350), colormap = :turbo)
```
Comment (fisa47) [inline src/output.jl]: Maybe make for any tracer, not only oxygen?

File: app/varna/plot.jl (hunk 1 above)
Comment (m-n-o) [reply app/varna/plot.jl]: Done

File: src/output.jl (hunk 2 above)
Comment (m-n-o) [reply src/output.jl]: Done.

**Pull request merged**